test/ret_test.exs

```diff
@@ -157,6 +161,147 @@ defmodule RetTest do
       assert 0 === count(HubRoleMembership, hub)
       assert 0 === count(RoomObject, hub)
     end
+
+    test "deletes account's avatars" do
+      {:ok, admin_account: admin_account} = create_admin_account("admin")
+      test_account = create_account("test")
+
+      [_avatar, avatar_owned_files] = create_avatar(test_account)
+
+      1 = count(Avatar, test_account)
+      7 = count(OwnedFile, test_account)
+      true = owned_files_exist?(avatar_owned_files)
+
+      assert :ok = Ret.delete_account(admin_account, test_account)
+
+      assert 0 === count(Avatar, test_account)
+      assert 0 === count(OwnedFile, test_account)
+      assert false === owned_files_exist?(avatar_owned_files)
+    end
+
+    test "listed avatars are reassigned to admin account" do
+      {:ok, admin_account: admin_account} = create_admin_account("admin")
+      target_account = create_account("target")
+
+      [avatar, avatar_owned_files] = create_avatar(target_account)
+      create_avatar_listing(avatar, target_account)
+
+      1 = count(Avatar, target_account)
+      7 = count(OwnedFile, target_account)
+      1 = count(AvatarListing, target_account)
+
+      0 = count(Avatar, admin_account)
+      0 = count(OwnedFile, admin_account)
+      0 = count(AvatarListing, admin_account)
+
+      assert :ok = Ret.delete_account(admin_account, target_account)
+
+      assert 0 === count(Avatar, target_account)
+      assert 0 === count(OwnedFile, target_account)
+      assert 0 === count(AvatarListing, target_account)
+
+      assert 1 === count(Avatar, admin_account)
+      assert 7 === count(OwnedFile, admin_account)
+      assert 1 === count(AvatarListing, admin_account)
+
+      assert true === owned_files_exist?(avatar_owned_files)
```

Comment:
This might read a little nicer as `assert owned_files_exist?(avatar_owned_files)`